crates/iota-crypto/src/simple.rs

```diff
@@ -467,7 +467,12 @@ mod keypair {
     }
 }
 
-#[cfg(test)]
+#[cfg(all(
+    test,
+    feature = "ed25519",
+    feature = "secp256k1",
+    feature = "secp256r1"
```

Comment:
I guess we could have put the specific features on the specific tests, otherwise tests are only run together

Reply:
Not a big deal though

Reply:
I did this initially, but it ends up in a big mess and at the end also doesn't really test much anymore, because all the tests contain parts of the other types